pylint/checkers/classes/scope_access_map.py

```diff
@@ -0,0 +1,31 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+
+import collections
+
+from pylint.checkers.utils import node_frame_class
```

Comment:
The function and class here should be part of `class_checker.py`. They aren't used anywhere else.